lib/utils/token.go

```diff
@@ -27,9 +27,10 @@ import (
 // from the disk if it looks like a path,
 // otherwise, treat it as a value
 func ReadToken(token string) (string, error) {
-	if !strings.HasPrefix(token, "/") {
+	if !strings.HasPrefix(token, "/") && !strings.HasPrefix(token, "./") {
```

Comment:
Would it make sense to add `../` too?

Reply:
I should have caught this earlier when validating the docs!

Thinking more carefully about this. Couple of other scenarios maybe worth considering:

1. If the token's in a different path (say with `~` at the immediate start of the path string). I could see that being the case.
2. The new Vault will mitigate some of these concerns since the token will be further secured and stored rather than shared via string or file. I wonder if it's better to fix the one or two spots in the docs and recommend the Vault? 

Happy to make that adjustment to the I think one or two places we currently use `./`.

Reply:
Homedir expansion is not trivial:

https://github.com/mitchellh/go-homedir/blob/master/homedir.go

I made the code more flexible to detect path present anywhere in the string to support local paths better.